### ISSUE TITLE
the inputsocket @timestamp problem

### ISSUE THREAD

**Issue by aantono** [tsaikd/gogstash#34]
When `input` socket module is being used and the payload is a valid JSON, the `@timestamp` field is not being properly initialized to `time.Now()`.  This results in the output `LogEvent` having a non-initialized `Timestamp` field.

Currently the result of the output has a `@timestamp` being this:

```
{
	"@timestamp": "0001-01-01T00:00:00Z",
	...
}
``` 

Using version 0.1.14

**Comment by aantono** [comment]
Proposed fix would be to change line https://github.com/tsaikd/gogstash/blob/master/input/socket/inputsocket.go#L153 to be:

```
msgChan <- logevent.LogEvent{Timestamp: time.Now(), Extra: jsonMsg}
```

**Comment by tsaikd** [comment]
@aantono Thanks for recommendation, it should be fixed in c9122e4150018ca4d200c8eec9514ee432596e96. Please try again.

**Comment by aantono** [comment]
@tsaikd Thanks a lot for a quick fix.  Just did a local build and verified that the `@timestamp` is now properly populated.  This issue can be closed, but I wonder if it would be a good idea to go through the other `input` modules to make sure that it is properly set everywhere.  It looks like `httplisten` also has the same issue as this one. (https://github.com/tsaikd/gogstash/blob/master/input/httplisten/httplisten.go#L102)

**Comment by tsaikd** [comment]
@aantono Thanks, I reviewed all input modules. Only httplisten module was vulnerable, and fixed in 0eab184225dd12c7ecdedfc4a4f88151ae6a6605.
